packages/mailer/src/Priority.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Tempest\Mail;
+
+enum Priority: int
+{
+    /**
+     * The highest priority.
```

Comment:
We don't really need these docblocks imo

Reply:
Yeah, I got lazy. Oops. I updated them to actually be useful.